server/server/api/views.py

```diff
@@ -564,13 +564,28 @@ def get(self):
         return data.batch(), 200
 
 class DictionaryFull(Resource):
-    def get(self):
+    def get(self, word=None):
         """
+        Sends list of dictionary entries to dictionary view
+        ---
+        responses:
+            dictionary_full:
+                type: array
+                properties:
+                    dictname:
+                        type: string
+                    word:
+                        type: string
+                    text:
+                        type: string
 
         """
+        if (word != None):
```

Comment:
`is not` should be used instead of `!=`
https://www.python.org/dev/peps/pep-0008/#programming-recommendations

Reply:
You're right. Don't ask me to do anything at 5 am ... my brain is not there and I mix up JS & python. Will change it. Thanks!